todo_app/templates/users.html

```diff
@@ -0,0 +1,52 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+    <div class="jumbotron bg-secondary text-white">
+        <h1 class="display-4">To-Do App</h1>
+    </div>
+    <div class="card border-dark text-dark bg-light mb-2">
+        <div class="card-body">
+
+            <form action="{{ url_for('index') }}" method="get">
+                <button type="submit" name="users-button" class="btn btn-outline-primary mr-1 mb-4">To-do list</button>
+            </form>
```

Comment:
Could have made this a link instead (aka an anchor, `<a>` tag) Just slightly simpler. You can still use those classes to make it look like a button rather than blue text.